README.md

```diff
@@ -37,51 +37,74 @@ Check out the [🌩 **Nimbus Documentation Hub**](https://experimenter.info) or
 1. Prerequisites
 
     On all platforms:
-    - Install [Docker](https://www.docker.com/)
-    - Install [Node](https://nodejs.org/en/) ^14.0.0
-
+    - Install [Node](https://nodejs.org/en/) ^14.0.0 <17.0 (recommended: 16.9.1)  
+    
     On Linux:
     - [Setup docker to run as non-root](https://docs.docker.com/engine/security/rootless/)
 
     On MacOS:
-      - [Install Docker Desktop](https://www.docker.com/products/docker-desktop)
-      - Adjust resource settings
-        - CPU: Max number of cores
-        - Memory: 50% of system memory
-        - Swap: Max 4gb
-        - Disk: 100gb+
-
-1.  Clone the repo
+      - Install [yarn](https://github.com/yarnpkg)
+
+            brew install yarn
+            
+      - Install [Docker](https://docs.docker.com/desktop/mac/install/)
+          - Adjust resource settings
+            - CPU: Max number of cores
+            - Memory: 50% of system memory
+            - Swap: Max 4gb
+            - Disk: 100gb+
+            
+1. Clone the repo
 
         git clone <your fork>
 
-1.  Copy the sample env file
+1. Copy the sample env file
 
         cp .env.sample .env
 
-1.  Set DEBUG=True for local development
+1. Set DEBUG=True for local development
 
         vi .env
 
-1.  Create a new secret key and put it in .env
+1. Create a new secret key and put it in .env
 
         make secretkey
 
-1.  Run tests
+	vi .env
+
+	```
+	...
+	SECRETKEY=mynewsecretkey
+	...
+	```
+
+1. Run tests
 
         make check
 
-1.  Setup the database
+1. Setup the database
 
         make refresh
 
+1. If using VSCode, configure workspace folders
+
+   - Add `app` folder to your workspace (File -> Add Folder to Workspace -> `path/to/experimenter/app`)
```

Comment:
You have to add both folders to workspace:

`/experimenter/`
`/experimenter/app`